src/einsteinpy/symbolic/ricci.py

```diff
@@ -166,9 +169,10 @@ def __init__(self, expression, syms, parent_metric=None):
             Raised when syms is not a list or tuple
 
         """
+        super(RicciScalar, self).__init__(
+            arr=arr, syms=syms, config=config, parent_metric=parent_metric
+        )
         self._order = 0
-        self._parent_metric = parent_metric
-        self._expr = expression
         if isinstance(syms, (list, tuple)):
             self.syms = syms
```

Comment:
remove the whole `if isinstance` code block. This is being taken care by `BaseRelativityTensor`